4-statistics/Task_excel.R

```diff
@@ -0,0 +1,95 @@
+# This script demonstrates how to use R for data analysis tasks that have been
```

Comment:
Would be good to get these to run in CI and to show the results, will have a think.